src/sampling.jl

```diff
@@ -5,7 +5,13 @@
 #
 ###########################################################
 
-using Random: Sampler, Random.GLOBAL_RNG
+using Random: Sampler
+
+@static if VERSION < v"1.3"
```

Comment:
```suggestion
if VERSION < v"1.3.0-DEV.565"
```
`@static` isn't necessary at the top level. Also I tracked down the commit that defined `default_rng` and got its version for maximum ~~pedantry~~ accuracy. Alternatively, we could use `isdefined(Random, :default_rng)` in place of a version check.

Reply:
Ah, my bad. I read a while ago about unnecessary `@static`s but still sometimes add them a bit too freely 😅 And thanks for looking up the commit, I had decided to just not care about the 1.3 dev versions 😉

Reply:
AFAIK it doesn't hurt to have it there, it just doesn't have any effect.

> thanks for looking up the commit

Any chance I can get to use Julia's `contrib/commit-name.sh`, I take